dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -85,7 +85,7 @@ class TeacherApplication < ApplicationBase
 
     before_save :save_partner, if: -> {form_data_changed? && regional_partner_id.nil? && !deleted?}
     before_save :update_user_school_info!, if: -> {form_data_changed?}
-    before_save :log_status, if: -> {status_changed?}
+    before_save :log_status, if: -> {status_changed? || form_data_changed?}
```

Comment:
Form data changes when a user works on their incomplete application. We will now log that they updated their application (via another `incomplete` status in the `status_log`) so that we can send mailers at the correct time based on when they last updated their application

Reply:
Would this queue an email for each form data change assuming a week has past? Like if I updated 10 fields and saved but did not complete it for a week, would I receive 10 reminder emails? Or is this just updating when the most recent 'incomplete' status was logged and the reminder email would only be triggered once there is a week gap since the most recent 'incomplete' log?

Reply:
Oh my bad sorry, I missed the logic that accounts for this in `incomplete_reminder.rb` during my first pass through. This looks great!

Reply:
Good question! I had the same concern